articulate/orson
# Fix control bar tab order

Fix the tab order of the buttons in the control bar. Previously, our flexbox ordering caused tabbing from the seekbar to jump all the way to the volume controls. This abandons the flexbox ordering & instead orders the elements explicitly. Controlbar children can still be overridden by a consumer.

### Test
* Ensure that when moving though control bar elements via tab, focus moves from left to right.
* Ensure that control bar options can be set via a prop `options={{ controlBar: { ... } }}`
